frontend/src/app/modules/interventions/details/details.component.ts

```diff
@@ -1,25 +1,17 @@
-import { Component, Inject, Input } from '@angular/core';
+import { Component, Input } from '@angular/core';
 import { Router } from '@angular/router';
 
 import { Intervention } from '@shared/domain/intervention.model';
-import { ComponentWithSubscriptions } from '@shared/components/base';
-import { GTM_CONTEXTS } from '@shared/google-tag-manager/gtm-contexts';
 
 @Component({
   selector: 'eko-intervention-details',
   templateUrl: './details.component.html',
   styleUrls: ['./details.component.scss'],
 })
-export class DetailsComponent extends ComponentWithSubscriptions {
+export class DetailsComponent {
   @Input() intervention: Intervention;
-  @Input() embedded?: boolean;
 
-  interventionDetailsGtmContext: string;
-
-  constructor(private router: Router, @Inject(GTM_CONTEXTS) gtmContexts) {
-    super();
-    this.interventionDetailsGtmContext = gtmContexts.interventionDetails;
-  }
+  constructor(private router: Router) {}
 
   navigateToEditView() {
     this.router.navigate(['interwencje', this.intervention.id, 'edytuj']);
```

Comment:
Maybe you can use here `EkoRoutePaths.Interventions` or create there a function which returns a path to edit screen given intervention id

Reply:
Good point! I used enums for that for now. If there will be a need to reuse those paths then we can refactor to functions :)